packages/e2e-test-utils/src/withUser.js

```diff
@@ -28,11 +29,15 @@ import { getCurrentUser, setCurrentUser } from './user';
 export default function withUser(username, password) {
   const currentUser = getCurrentUser();
 
-  /* eslint-disable jest/require-top-level-describe, require-await */
-  beforeAll(async () => setCurrentUser(username, password));
+  beforeAll(async () => {
+    await setCurrentUser(username, password);
 
+    // Disable cross-origin isolation by default as it causes issues in Firefox.
+    await toggleVideoOptimization(false);
```

Comment:
Can this be removed if we bump up the same function call in `beforeAll()` in `bootstrap.js`?

Reply:
Since video optimization is a user-level setting, whenever we log in as a different user, we'd need to check whether the setting is disabled.